### PR TITLE
Save the JVM error log from product tests

### PR DESCRIPTION
**Pull request by cawallin**

### CODE REVIEW

**Comment by cawallin** [comment]
This should help in cases like https://github.com/prestodb/presto/issues/8084 when we want to be able to inspect JVM error logs.

File: presto-product-tests/conf/docker/common/standard.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ services:
     image: 'teradatalabs/centos6-java8-oracle:${DOCKER_IMAGES_VERSION}'
     volumes:
       - ../..:/docker/volumes/conf
+      - ../../target:/docker/volumes/logs
```
Comment (ArturGajowy) [inline presto-product-tests/conf/docker/common/standard.yml]: can we make it `../../target/logs`?

File: .travis.yml [hunk 2]
```diff
@@ -138,6 +138,9 @@ after_script:
       --include='**/' \
       --include='**/surefire-reports/**.xml' \
       --include='**/surefire-reports/emailable-report.html' \
+      --include='**/test-reports/junitreports/**.xml' \
+      --include='**/test-reports/emailable-report.html' \
```
Comment (ArturGajowy) [inline .travis.yml]: these two lines are an unrelated change, please split into another commit

Comment (ArturGajowy) [inline .travis.yml]: Also - just to make sure - is it needed because that's where the product tests put test reports?

Comment (cawallin) [inline .travis.yml]: yup, we weren't saving the logs for product test reports

File: presto-product-tests/conf/docker/common/standard.yml (hunk 1 above)
Comment (cawallin) [inline presto-product-tests/conf/docker/common/standard.yml]: does docker create the directory if it doesn't exist? I didn't see a good place to create that logs directory in target, so I left it just target.

Comment (ArturGajowy) [inline presto-product-tests/conf/docker/common/standard.yml]: I don't think so, we probably would need plug in s/w around here:
```
function run_product_tests() {
  local REPORT_DIR="${PRODUCT_TESTS_ROOT}/target/test-reports"
  rm -rf "${REPORT_DIR}"
  mkdir -p "${REPORT_DIR}"
```
But I think additional 3 lines of code are not worth the `../../target/logs/` comment, let's drop it :)

**Comment by ArturGajowy** [comment]
Please just split into the two commits :)

File: presto-product-tests/conf/presto/etc/jvm.config [hunk 3]
```diff
@@ -19,3 +19,4 @@
 -Duser.timezone=UTC
 -Xdebug 
 -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005
+-XX:ErrorFile=/docker/volumes/logs/product-tests-presto-jvm-error-file.log
```
Comment (findepi) [inline presto-product-tests/conf/presto/etc/jvm.config]: Can't we simply capture stderr?

Comment (cawallin) [inline presto-product-tests/conf/presto/etc/jvm.config]: No, we want the additional file that the JVM produces. It defaults to being in /var/presto/hs_err_pid5.log, and is lost when the docker container stops (which happens immediately after Presto terminates).

**Comment by cawallin** [comment]
@ArturGajowy done

**Comment by kokosing** [comment]
Rebased, I am going to merge this once travis passes.
